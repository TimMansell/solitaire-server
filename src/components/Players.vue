<template>
  <div>
    Games:
    <Counter data-test="player-count" :number="playerCount" />
  </div>
</template>

<script>
import { mapGetters } from 'vuex';
import Counter from '@/components/Counter.vue';

export default {
  name: 'Players',
  components: {
    Counter,
  },
  computed: {
    ...mapGetters(['playerCount']),
  },
};
</script>
