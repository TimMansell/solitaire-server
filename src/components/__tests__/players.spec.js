import { shallowMount } from '@vue/test-utils';
import Players from '@/components/Players.vue';

describe('Players.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Players, {
      computed: {
        playerCount: () => 1,
      },
    });

    expect(wrapper).toMatchSnapshot();
  });
});
