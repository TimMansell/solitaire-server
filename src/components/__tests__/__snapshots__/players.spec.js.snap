// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Players.vue matches snapshot 1`] = `
<div>
  Players:
  <counter-stub number="1" data-test="player-count"></counter-stub>
</div>
`;
